lib/Twig/Template.php

```diff
@@ -419,7 +419,13 @@ protected function getAttribute($object, $item, array $arguments = array(), $typ
                         $message = sprintf('Key "%s" for array with keys "%s" does not exist', $arrayItem, implode(', ', array_keys($object)));
                     }
                 } elseif (Twig_Template::ARRAY_CALL === $type) {
-                    $message = sprintf('Impossible to access a key ("%s") on a %s variable ("%s")', $item, gettype($object), $object);
+                    if (null === $object) {
+                        $message = sprintf('Impossible to access a key ("%s") on a %s variable', $item, gettype($object));
```

Comment:
Here you know that `$object = null`. Then you can write it directly instead of relying on `gettype`.

```php
$message = sprintf('Impossible to access a key ("%s") on a null variable', $item);
```